### PR TITLE
Review 

### PR DESCRIPTION
**Pull request by vjspranav**

### CODE REVIEW

File: experiment/simulation/index.html [hunk 1]
```diff
@@ -1,13 +1,275 @@
...
+          id="last-container"
+          fill="url(#half_grad16)"
+          fill="#00ccff"
+          opacity="1.00"
+          d=" M 692.99 334.52 C 756.84 334.53 820.68 334.51 884.52 334.53 C 884.41 353.34 884.62 372.16 884.43 390.98 C 820.62 390.85 756.81 390.87 693.01 390.98 C 692.94 372.16 693.01 353.34 692.99 334.52 Z"
+        />
+      </g>
+    </svg>
+    <!-- Js files -->
+    <script src="./js/main.js"></script>
```
Comment (raj-vlabs) [inline experiment/simulation/index.html]: Would you consider including main.js after the other scripts, which main,js may be depending on?

File: experiment/simulation/js/main.js [hunk 2]
```diff
@@ -1 +1,130 @@
...
+  "tap2-pipe-after",
+  "tap2-stream-before",
+  "last-container",
+];
+
+let asyncMove = async (id, curPosition = 0, finalPosition = 1) => {
+  let path = document.getElementById(id);
+  let flags = [true, true, true, true, true, true, true];
+  while (true) {
+    speed2 = document.getElementById("water-flow").value;
```
Comment (raj-vlabs) [inline experiment/simulation/js/main.js]: speed2 can be given a more meaningful name

File: experiment/simulation/index.html (hunk 1 above)
Comment (vjspranav) [reply experiment/simulation/index.html]: For this experiment in particular, main.js is not dependent on any other script. Though if still needed we can include it at the end

**Review by raj-vlabs — APPROVED** [review]
Please merge the changes to the main branch.

**Pull request merged**